test/tests/autoinc1.sh

```diff
@@ -40,7 +40,7 @@ EOF
 
   fsed "s/REPLACE_VALUE/$1/g" "${PROJECT_PATH}/src/lib.rs"
 
-  run_test cargo run publish --project-path "$PROJECT_PATH" --clear-database -d -S
+  run_test cargo run publish --project-path "$PROJECT_PATH" --clear-database --skip_clippy
```

Comment:
It would probably be better to use an env variable like `PUBLISH_ARGS` or something instead of just blanketing `--skip_clippy` here, but I'm fine with this